overlay/ipc_utils.h

```diff
@@ -0,0 +1,13 @@
+// Copyright 2015-2022 The Mumble Developers. All rights reserved.
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file at the root of the
+// Mumble source tree or at <https://www.mumble.info/LICENSE>.
+
+#ifndef MUMBLE_OVERLAY_IPC_UTILS_H__
+#define MUMBLE_OVERLAY_IPC_UTILS_H__
+
+char *getRuntimePath__();
+
+char *getAndCreateOverlayPipePath__();
```

Comment:
why the double-underscores in the name? Usually that is reserved for internal library functions and should be avoided in userspace land

Reply:
I used this notation to distinguish it from the functions in `src/mumble/IPCUtils[.h, .cpp]`. They wrap the C functions in C++.
If you have better a idea, please tell me! I'm short of creativity hahahaha